locale/en/knowledge/javascript-conventions/what-is-the-arguments-object.md

```diff
@@ -24,41 +24,63 @@ property of `arguments` is callee. This always refers to the function
 currently being executed. Here's an example that illustrates the
 properties of `arguments`.
 
-    var myfunc = function(one) {
-      arguments.callee === myfunc;
-      arguments[0] === one;
-      arguments[1] === 2;
-      arguments.length === 3;
-    }
+```js
+const myfunc = function(one) {
+  arguments.callee === myfunc;
+  arguments[0] === one;
+  arguments[1] === 2;
+  arguments.length === 3;
+}
 
-    myfunc(1, 2, 3);
+myfunc(1, 2, 3);
+```
 
 This construct is very useful and gives javascript functions a lot of
 flexibility. But there is an important gotcha. The `arguments` object
 behaves like an array, but it is not an actual array. It does not have
 Array in its prototype chain and it does not respond to any array
 methods, e.g. `arguments.sort()` raises a TypeError. Instead you need to
-copy the values into a true array first. Since a normal for loop
-works, this is pretty easy.
+copy the values into a true array first. Since a normal `for` loop
+works, this is pretty easy, or if you are working with ES6 and above `Array.from()` makes this transfer even easier.
 
-    var args = [];
-    for(var i = 0; i < arguments.length; i++) {
-      args.push(arguments[i]);
-    }
+```js
+//  for ES5 and below
+const args = [];
+for(let i = 0; i < arguments.length; i++) {
+  args.push(arguments[i]);
+}
+
+// for ES6 and above
```

Comment:
Maybe we should push this as the default for current Node.js versions (and only mention the pre-ES6 variants briefly). What do you think?

Reply:
Thanks, @fhemberger for the advice will update both the sections :)